lib/db.js

```diff
@@ -345,6 +345,23 @@ module.exports = function (
       )
   }
 
+  DB.prototype.keyFetchTokenWithVerificationStatus = function (id) {
+    log.trace({ op: 'DB.keyFetchTokenWithVerificationStatus', id: id })
+    return this.pool.get('/keyFetchToken/' + id.toString('hex') + '/verified')
+      .then(
+        function (body) {
+          var data = bufferize(body)
+          return KeyFetchToken.fromId(id, data)
+        },
+        function (err) {
+          if (isNotFoundError(err)) {
+            err = error.invalidToken()
+          }
+          throw err
+        }
+      )
+  }
```

Comment:
@vbudhram you need to also update `createKeyFetchToken` in this file, to accept the new `tokenVerificationId` parameter.  Otherwise it just silently ignores it and all keyFetchTokens are created verified.